packages/js-core/src/website/lib/actions.ts

```diff
@@ -1,36 +1,22 @@
-import { NetworkError, Result, okVoid } from "../../shared/errors";
+import { InvalidCodeError, NetworkError, Result, err, okVoid } from "../../shared/errors";
 import { Logger } from "../../shared/logger";
 import { WebsiteConfig } from "./config";
 import { triggerSurvey } from "./widget";
 
 const logger = Logger.getInstance();
 const websiteConfig = WebsiteConfig.getInstance();
 
-export const trackAction = async (name: string): Promise<Result<void, NetworkError>> => {
-  const {
-    state: { surveys = [] },
-  } = websiteConfig.get();
-
-  // if surveys have a inline triggers, we need to check the name of the action in the code action config
-  surveys.forEach(async (survey) => {
-    const { inlineTriggers } = survey;
-    const { codeConfig } = inlineTriggers ?? {};
-
-    if (name === codeConfig?.identifier) {
-      await triggerSurvey(survey);
-      return;
-    }
-  });
-
-  logger.debug(`Formbricks: Action "${name}" tracked`);
+export const trackAction = async (name: string, alias?: string): Promise<Result<void, NetworkError>> => {
+  alias = alias || name;
```

Comment:
please don't directly modify a function parameter as it is against immutability, you can use: 
`const alias = aliasParameter || name`